mcs/class/corlib/corert/RuntimeImports.cs

```diff
@@ -0,0 +1,18 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime
+{
+    public static class RuntimeImports
+    {
+        [DllImport("__Internal", ExactSpelling = true)]
+        internal static extern unsafe void _ecvt_s(byte* buffer, int sizeInBytes, double value, int count, int* dec, int* sign);
```

Comment:
what does this end up calling to?

Reply:
@akoeplinger https://msdn.microsoft.com/en-us/library/ehz2dset.aspx (c run-time library) - it's needed only for Windows - https://github.com/dotnet/corert/blob/e79755bebe745c1229df8e5ba9a20b4fa3fc950d/src/System.Private.CoreLib/src/System/Number.Windows.cs#L27 and https://github.com/dotnet/corert/blob/df8d48011ebc11c5c97b8384275d40ef4f95388e/src/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs#L1015 but I am not quite sure why we can't just use `snprintf` icall on all platforms